vignettes/pr_review_guidance.Rmd

```diff
@@ -0,0 +1,79 @@
+---
+title: "Pull Request Review Guidance"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{pr_review_guidance}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Fo a pull request (PR) to be merged it needs to pass the automated `R CMD check` and `lintr` workflows at a minimum. In addition the PR reviewer should make sure that
```

Comment:
- Typo 'Fo' instead of 'For'
- Should we explicitly say how to run the`R CMD check`and lintr? this would have helped me at the start :) devtools:check(), lintr::lint/lint_pakage?